linera-storage-service/src/child.rs

```diff
@@ -49,14 +51,14 @@ impl StorageServiceSpanner {
     }
 
     pub async fn run_service(&self) -> Result<StorageServiceGuard> {
-        self.wait_deadness().await?;
+        self.wait_absence().await?;
```

Comment:
```suggestion
        self.wait_absence().await?;
```

Maybe `await_absence` or `wait_for_absence`?

Reply:
Ok for `wait_for_absence`.